x-pack/plugins/cloud_security_posture/public/pages/benchmarks/benchmarks_table.tsx

```diff
@@ -68,6 +68,12 @@ const BENCHMARKS_TABLE_COLUMNS: Array<EuiBasicTableColumn<Benchmark>> = [
     truncateText: true,
     sortable: true,
   },
+  {
+    field: 'rules',
+    name: TABLE_COLUMN_HEADERS.ACTIVE_RULES,
+    truncateText: true,
+    render: (rules: Benchmark['rules']) => `${rules.enabled} of ${rules.all}`,
```

Comment:
this needs to be a `FormattedMessage` with smth like `{enabled} of {all}`  then `values: {{ enabled, all }}`